bin/eslint.js

```diff
@@ -7,7 +7,12 @@ var exitCode = 0,
 
 if (useStdIn) {
     process.stdin.pipe(concat({ encoding: "string" }, function(text) {
-        exitCode = cli.execute(process.argv, text);
+        try {
+            exitCode = cli.execute(process.argv, text);
+        } catch (ex) {
+            console.error(ex.message);
+            exitCode = 1;
+        }
```

Comment:
Wouldn’t this error also end up in the `error` event of the stream? Anyway it is always a good idea to add an error handler to every error event (https://github.com/maxogden/concat-stream#error-handling).